### PR TITLE
Fixes, Tests, & Enhancements

### PR DESCRIPTION
**Pull request by E-B-Smith**
* Fixed some image bugs
* Added a test a composed image for testing
* Small fixes and formatting.

### CODE REVIEW

File: Sources/AFIcon.swift [hunk 1]
```diff
@@ -33,7 +33,11 @@ enum ColorShade: CaseIterable {
 }
 
 enum AFIcon {
-    static func named(_ name: String, scheme: ColorScheme, size: CGSize = CGSize(width: 24.0, height: 24.0)) -> UIImage {
+    static func named(
+        _ name: String,
+        scheme: ColorScheme,
+        size: CGSize = CGSize(width: 24.0, height: 24.0)
+    ) -> UIImage {
```
Comment (E-B-Smith) [inline Sources/AFIcon.swift:40]: Wrapped for legibility.

File: Sources/AFIcon.swift [hunk 2]
```diff
@@ -84,24 +88,26 @@ extension UIColor {
         case (.custom(_, _, let dark), .dark): return dark
         }
 
-        return UIColor(fromHex: hexCode)
+        return UIColor(hex: hexCode)
```
Comment (E-B-Smith) [inline Sources/AFIcon.swift:91]: Tersified.

File: Sources/UIImage+Effects.swift [hunk 3]
```diff
@@ -45,18 +45,19 @@ extension UIImage {
        - size:      The size of the image.
      - Returns:     Returns an image with a solid color of the given size.
     */
-    public static func image(color: UIColor, size: CGSize) throws -> UIImage {
-        UIGraphicsBeginImageContextWithOptions(size, false, 0.0)
+    public convenience init(color: UIColor, size: CGSize) throws {
+        let scale = UIScreen.main.scale
+        UIGraphicsBeginImageContextWithOptions(size, false, scale)
```
Comment (E-B-Smith) [inline Sources/UIImage+Effects.swift:50]: The image scale was ignored, making the image size wrong in some cases.

File: Sources/ViewController.swift [hunk 4]
```diff
@@ -77,22 +77,34 @@ class ViewController: UIViewController {
...
+                (UIColor(red: 243/255, green: 187/255, blue: 75/255, alpha: 1.0), .gray),
                 (UIColor(red: 206/255, green: 72/255, blue:69/255, alpha: 1.0), .black),
-            ], tolerance: 0.01)
-        let labeledImage = LabeledImage(text: "Enhanced", image: image!)
+            ], tolerance: 0.015)
+        let labeledImage = LabeledImage(text: "Color Swapped", image: image!)
+        return labeledImage
+    }
 
+    private func composedLogoView() -> UIView {
```
Comment (E-B-Smith) [inline Sources/ViewController.swift:87]: Added a "Composed" image made up of a series of images strung together.

File: Tests/ColorsTests.swift [hunk 5]
```diff
@@ -94,4 +94,10 @@ class ColorsTests: XCTestCase {
         let color = colorFromPixel(pixel)
         assertColorsEqual(color, kTestColor)
     }
+
+    func testDistance() {
+        XCTAssertEqual(distance(0xffffffff, [0, 0, 0, 0]), 1.0)
+        XCTAssertEqual(distance(0xffffffff, [255, 255, 255, 255]), 0.0)
+        XCTAssertEqual(distance(0x7f7f7f7f, [255, 255, 255, 255]), 0.25, accuracy: 0.002)
+    }
```
Comment (E-B-Smith) [inline Tests/ColorsTests.swift:102]: Added another test.

**Review by stephanie-lee — APPROVED** [review]

**Pull request merged**